src/Robo/Tasks/Drush.php

```diff
@@ -0,0 +1,267 @@
+<?php
+
+namespace Acquia\Blt\Robo\Tasks;
+
+use Robo\Task\CommandStack;
+use Robo\Contract\VerbosityThresholdInterface;
+use Robo\Common\CommandArguments;
+
+/**
+ * Runs Drush commands in stack. You can use `stopOnFail()` to point that stack should be terminated on first fail.
+ *
+ * ``` php
+ * $this->taskDrush()
+ *     ->drush('updb')
+ *     ->drush('cr')
+ *     ->run();
+ * ```
+ */
+class Drush extends CommandStack
+{
+  use CommandArguments;
+
+  /**
+   * Site alias to prepend to each command.
+   *
+   * @var string
+   */
+  private $alias;
+
+  /**
+   * Directory to execute the command from.
+   *
+   * @var string
+   *
+   * @see ExecTrait::$workingDirectory
+   */
+  private $dir;
+
+  /**
+   * Site uri to append uri option to each command.
+   *
+   * @var string
+   */
+  private $uri;
+
+  /**
+   * Assume 'yes' or 'no' to all prompts.
+   *
+   * @var string|bool
+   */
+  private $assume;
+
+  /**
+   * Attach tty to process for interactive input.
+   *
+   * @var bool
+   *
+   * @see ExecTrait::$interactive
+   */
+  private $passthru;
+
+  /**
+   * Indicates if command output should be printed.
+   *
+   * @var bool
+   *
+   * @see ExecTrait::isPrinted
+   */
+  private $logOutput;
+
+  /**
+   * Indicates if the command output should be verbose.
+   *
+   * @var bool
+   */
+  private $verbose;
+
+  /**
+   * @todo Figure out how to fetch config from constructor to avoid this.
+   *
+   * @var bool
+   */
+  private $defaultsInitialized;
+
+  /**
+   * Runs the given drush command.
+   *
+   * @param string $command
+   *
+   * @return $this
+   */
+  public function drush($command) {
+    // @todo Figure out how to fetch config from constructor to avoid this.
+    if (!$this->defaultsInitialized) {
+      $this->init();
+    }
+
+    if ($this->alias) {
+      $command = "@{$this->alias} {$command}";
+    }
+
+    if (!empty($this->uri)) {
+      $this->option('-l', $this->uri);
```

Comment:
No problem, I did it because it appeared cleaner programatically. I didn't see support through the `option` function to pass in an `=` to separate the option name and value as opposed to the space that they default to.